Tests/TestingApps/PaywallsTester/PaywallsTester/Products.storekit

```diff
@@ -169,7 +169,7 @@
     }
   ],
   "version" : {
-    "major" : 2,
+    "major" : 3,
```

Comment:
Is this going to break anything? Is this because of Xcode 16 or something else? 😅

Reply:
I'm using Xcode 15, I'm not sure why it made this change, but this is used solely internally for testing, so I don't think there is any major risk here?